CONTRIBUTING.md

```diff
@@ -208,6 +224,17 @@ $ poetry run behave integration-tests --include <feature-file>
 $ poetry run behave integration-tests -n "<scenario-name>"
 ```
 
+## Add the ipdb debugger
```

Comment:
This is not required so maybe you can tell users why they might want to **optionally** add this debugger?

Reply:
@zaro0508 I have updated the docs to be less prescriptive about which debugger to use. Otherwise, I think contributors can be expected to already know what the debugger is for?